chirpstack-concentratord-sx1302/src/cmd/root.rs

```diff
@@ -45,13 +45,26 @@ pub fn run(
     );
 
     // get concentrator eui
-    let gateway_id = concentrator::get_eui().unwrap();
+    let mut gateway_id = concentrator::get_eui().unwrap();
```

Comment:
There is some unneeded complexity here. The same can be achieved with something like this (untested, but it should give an idea):

```rust
let gateway_id = if let Some(gateway_id) = config.gateway.gateway_id_bytes {
    gateway_id
else {
    concentrator::get_eui().unwrap()
};
```

Reply:
Done. Thanks for the insight.